metadata-ingestion/src/datahub/ingestion/source/fivetran/fivetran.py

```diff
@@ -475,11 +474,14 @@ def _get_connector_workunits(
         yield datajob
 
         # Map Fivetran's job/sync history entity with Datahub's data process entity
-        if len(connector.jobs) >= MAX_JOBS_PER_CONNECTOR:
+        if (
```

Comment:
Correct me if I am wrong, but if we are already using `self.config.fivetran_log_config.max_jobs_per_connector` in the query used to retrieve the jobs, then this condition will be never met, isn't that right?